src/species_advance/standard/pipeline/advance_p_pipeline_v8.cc

```diff
@@ -80,8 +80,11 @@ advance_p_pipeline_v8( advance_p_pipeline_args_t * args,
   // Determine which accumulator array to use.
   // The host gets the first accumulator array.
 
-  a0 += ( 1 + pipeline_rank ) *
-        POW2_CEIL( (args->nx+2)*(args->ny+2)*(args->nz+2), 2 );
+  if ( pipeline_rank != n_pipeline )
+  {
```

Comment:
Does this fix a long standing v8 bug? Or something else?

Reply:
Actually, it does not fix a bug because the vector intrinsics versions will not be called with pipeline_rank == n_pipeline.  It provides an extra layer of safety against that possibility and provides some uniformity between the vector intrinsics versions and the reference implementation.